docs/_static/policy/v1.yaml

```diff
@@ -0,0 +1,133 @@
+openapi: 3.1.0
+info:
+  title: Policy backend API specification
+  version: 0.1.0
+servers:
+  - url: "http://localhost:1323"
+paths:
+  /{did}/presentation_definition:
+    parameters:
+      - name: did
+        in: path
+        description: URLEncoded DID.
+        required: true
+        example: did:web:example.com:1
+        schema:
+          type: string
+      - name: scope
+        in: query
+        description: |
+          This is the scope used in the OpenID4VP authorization request.
+          It is a space separated list of scopes.
+        required: true
+        schema:
+          type: string
+    get:
+      summary: Returns a presentation definition for the given DID and scope.
+      description: |
+        The DID is used for tenant selection. Not all tenants will probably support the same scopes.
+        The scope is used as selection criteria for the presentation definition.
+        It could be the case that the presentation definition is not found.
+        In that case the response will be 201 with an empty body.
+      operationId: "presentationDefinition"
+      tags:
+        - policy
+      responses:
+        "200":
+          description: |
+            DID has been found and the scope is supported.
+            If the scope is supported but no presentation definition is required, the response will be 200 with a presentation definition without any input descriptors.
```

Comment:
what would the use case for this be?

Reply:
public service, like a directory service